src/components/modals/data-table-modal.js

```diff
@@ -20,74 +20,114 @@
 
 import React, {Component} from 'react';
 import styled from 'styled-components';
-import window from 'global/window';
 
 import {ALL_FIELD_TYPES} from 'constants/default-settings';
 import FieldToken from 'components/common/field-token';
 import DatasetLabel from 'components/common/dataset-label';
 import {Clock} from 'components/common/icons/index';
+import {Table, Column} from 'react-virtualized';
 
 // Breakpoints
 import {media} from 'styles/media-breakpoints';
-
-const ReactDataGrid = window.navigator ? require('react-data-grid/dist/react-data-grid.min') : null;
-
-let shouldPreventScrollBack = false;
-
-if (window.navigator && window.navigator.userAgent) {
-  const {navigator} = window;
-  // Detect browsers
-  // http://stackoverflow.com/questions/5899783/detect-safari-using-jquery
-  const isMac = navigator.userAgent.match(/Macintosh/);
-  const is_chrome = navigator.userAgent.indexOf('Chrome') > -1;
-  const is_safari = navigator.userAgent.indexOf('Safari') > -1;
-  const is_firefox = navigator.userAgent.indexOf('Firefox') > -1;
-
-  // prevent chrome scroll back
-  shouldPreventScrollBack = isMac && (is_chrome || is_safari || is_firefox);
-}
+import AutoSizer from 'react-virtualized/dist/commonjs/AutoSizer'
```

Comment:
The missing piece in this PR is the dynamic size of the table, which autosizer may provide but i am trying to see if there are other options

Reply:
Removed Autosizer completely, disregard this one;